packages/semver2/semver2.1.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "info@dividat.com"
+authors: ["Dividat AG"]
+synopsis: "Semantic version handling for OCaml"
+license: "MIT"
+homepage: "https://github.com/dividat/ocaml-semver"
+bug-reports: "https://github.com/dividat/ocaml-semver/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.2.0"}
```

Comment:
```suggestion
  "dune" {>= "1.2.0"}
```
Dune cannot be tagged as build dependency for forward compatibility reasons

Reply:
Can you port this change upstream?

Reply:
Thanks for the modification!

Upstream change: https://github.com/dividat/ocaml-semver/commit/587a3c5696b8bcb973ec0fb1138d821239c7ce5a